### DISCUSSION TITLE
QField: "Preferred transform not available", where to place TIF file?

### DISCUSSION THREAD

**Discussion by dto** [opengisch/QField#4115] [Q&A]
![Screenshot_20230402-112546](https://user-images.githubusercontent.com/22819/229363106-fe589300-1395-4bbc-86f1-87bf049855bf.png)

Hi. I am receiving the error message shown in the screenshot: "the preferred transform is not available". The file it specifies is https://cdn.proj.org/us_noaa_nehpgn.tif   which I downloaded. I tried using "import dataset" on it, and this gives no error itself, but does not resolve the original "transform not available" error. I tried putting the TIF file in the ZIP and opening the project from the zip, but I still get the error. My question is: where on the device does this TIF file go so that the transform will be available to QField?

**Comment by lucienicolier** [comment]
Hi @dto ,

Is this doc can help you? [Altitude correction / vertical grid shift](https://docs.qfield.org/how-to/gnss/#altitude-correction-vertical-grid-shift)

**Reply by dto** [reply]
Thank you, I used an android file manager to place the .TIF file in the correct location and no longer get the error message :)
